core/rpc/json/user/commands.go

```diff
@@ -22,13 +22,13 @@ type VersionRequest struct{}
 
 // SchemaRequest contains the request parameters for MethodSchema.
 type SchemaRequest struct {
-	DBID string `json:"dbid,omitempty"`
+	DBID string `json:"dbid"`
 }
 
 // AccountRequest contains the request parameters for MethodAccount.
 type AccountRequest struct {
-	Identifier types.HexBytes `json:"identifier,omitempty"`
-	Status     *AccountStatus `json:"status,omitempty"` // Mapped to URL query parameter `status`.
+	Identifier types.HexBytes `json:"identifier" desc:"account identifier"`
+	Status     *AccountStatus `json:"status,omitempty" desc:"blockchain status (confirmed or unconfirmed)"` // Mapped to URL query parameter `status`.
```

Comment:
Oh, I was contemplating pulling parameter descriptions from the field tags like this.  I think I will remove for now.